test/cluster_tls_test.go

```diff
@@ -117,7 +117,7 @@ func TestClusterTLSInsecure(t *testing.T) {
 	defer srvA.Shutdown()
 
 	l := &captureTLSError{ch: make(chan struct{}, 1)}
-	srvA.SetLogger(l, false, false)
+	srvA.SetLogger(l, true, false)
```

Comment:
You need to modify the test in the `/test` package that uses this logger: `TestClusterTLSInsecure`.